ci/test/resources/config.yaml

```diff
@@ -0,0 +1,4 @@
+principals:
```

Comment:
What does this file do?

Reply:
This tells `create_certs.py` which applications to make certificates for and what the format of the kubernetes secret should be. JSON is used for all of the `aiohttp` services and `nginx` is for nginx proxies. This mirrors `tls/config.yaml`, I just didn't want to throw hello in there with the real services.